### PR TITLE
Update to handle 'cma parameter' configured events.

### PR DESCRIPTION
**Pull request by Jkovarik**
This PR addresses changes from https://github.com/nasa/cumulus/pull/1182 in Cumulus Core as well as nasa/cumulus-message-adapter#58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,16 @@ The Cumulus Message adapter for python provides one method:
...
     above)
   * `cumulus_message` - the event passed by Lambda, and should be a Cumulus
-    Message
+    Message, or a CMA parameter encapsulated message: 
+    
+```json
+{
+  'cma': {
+    'event': '<cumulus message object>
+    'SomeCMAConfigKey': '<CMA configuration object>
```
Comment (nemreid) [inline README.md]: Invalid JSON, use double quotes.

File: cumulus_logger.py [hunk 2]
```diff
@@ -39,19 +39,24 @@ def __getExceptionMessage(self, **kwargs):
             exceptionInfo = kwargs['exc_info'] if isinstance(kwargs['exc_info'], tuple) else sys.exc_info()
             exceptionStr = ' ' + logging.Formatter().formatException(exceptionInfo)
         return exceptionStr
+    
+    def __getEvent(self, event):
```
Comment (nemreid) [inline cumulus_logger.py]: This seems like a bad name for a function that fetches the execution name.

Comment (Jkovarik) [reply cumulus_logger.py]: 80b26bb

File: README.md (hunk 1 above)
Comment (Jkovarik) [reply README.md]: 87dc992

**Pull request merged**